javascript/modal.js

```diff
@@ -0,0 +1,156 @@
+const openModalButtons = document.querySelectorAll('[data-modal]');
+const closeModalButtons = document.querySelectorAll('[data-close-button]');
+const overlay = document.getElementById('overlay');
+const modalTitle = document.querySelector('.title');
+const modalImage = document.querySelector('.modal-img');
+const modalDescription = document.querySelector('.modal-desc p');
+const modalLangs = document.querySelector('.modal-categories');
+const liveBtn = document.querySelector('.live-btn');
+const codeBtn = document.querySelector('.code-btn');
+
+const portfolio = [
+  {
+    name: 'Tonic',
+    image: '../images/popup-tonic.svg',
+    description: ` Lorem Ipsum is simply dummy text of the printing and
+              typesetting industry. Lorem Ipsum has been the industry's
```

Comment:
- [ ] I noticed that your projects section in the `index.html` is still being hardcoded 👀. Please take advantage of your `portfolio` array and display your projects dynamically. Your HTML code for the projects list should be removed and instead added via JS. The advantage of this approach is that you have a single source of truth for your projects. If you decide to add a new project or delete an old project in the future, you only have to make the changes in your `portfolio` array and the changes will be seen everywhere. 🙏